BUILD

```diff
@@ -0,0 +1,93 @@
+Building eudev
+===================
+
+The following instructions provide the steps needed to acquire and build the eudev
+software on the Debian, Devuan, and compatible Linux distributions.
+
+Please note that command examples in this document use the following convention. Commands
+prefixed by a "$" symbol are to be run as a regular, user. Commands with a "#" prefix
+need to be run as the root user. If your user has sudo access then sudo may be
+used instead of logging into the root account.
+
+
+1. Installing dependencies
+============================
+
+The first thing we need to do is install the dependencies required to build eudev. 
+This can be accomplished by running the following two commands:
+
+# apt-get update
+
+# apt-get install build-essential pkg-config docbook-xsl\
+          docbook-xml m4 automake autoconf intltool gperf\
+          gnu-efi libcap-dev libselinux1-dev libacl1-dev\
+          libkmod-dev libblkid-dev gobject-introspection\
+          libglib-object-introspection-perl xsltproc 
+
+
+
+2. Downloading the source code
+================================
+
+There are two common approaches to downloading the project's source code. The first
+is to visit the eudev GitHub releases page (https://github.com/eudev-project/eudev/releases)
+and downloading the latest tar.gz archive file. Then unpack the downloaded archive. For
+example, if we downloaded eudev version 3.2.10 we could unpack the archive using the
+following command:
+
+$ tar xf eudev-3.2.10.tar.gz
+
+
+Alternatively, we can download the latest source code by using git. This is probably the
+better option for developers. Here we install the git source control program and
+use it to download the latest eudev source code:
+
+# apt-get install git
```

Comment:
IMO, no need to explain how to install git, up to you whether to keep that or not